src/main/java/com/hack/stock2u/chat/controller/ReservationController.java

```diff
@@ -0,0 +1,44 @@
+package com.hack.stock2u.chat.controller;
+
+import com.hack.stock2u.chat.dto.request.ReservationRequestDto;
+import com.hack.stock2u.chat.dto.response.ReservationResponse;
+import com.hack.stock2u.chat.service.ReservationService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import javax.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "채팅방 API")
```

Comment:
[변경 요청]
`@Tag(name = "채팅방 API")`

```suggestion
@Tag(name = "C. 채팅방 API")
```

C. 접두사 붙여주시면 문서에서 alphabetically 하게 정렬됩니다!